# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParentFactory.cpp
# Commit: b3fc2ce9d8cf
# Full Hash: b3fc2ce9d8cfd1ad2f7ca48d484cf060dcee0bc9
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2023-02-07 16:42:31
# Regressor Bug: 1809064
# File Overlap Count: 1
# Description:
#   Bug 1809064 - Expose FileSystemDataManager::mIOTarget; r=dom-storage-reviewers,jari
#   
#   This patch also fixes the misnamed getter for mIOTaskQueue.
#   
#   Depends on D166344
# ==============================================================================

diff -r e0d6b8852f52 -r b3fc2ce9d8cf dom/fs/parent/FileSystemManagerParentFactory.cpp
--- a/dom/fs/parent/FileSystemManagerParentFactory.cpp	Tue Feb 07 08:27:06 2023 +0000
+++ b/dom/fs/parent/FileSystemManagerParentFactory.cpp	Tue Feb 07 08:27:07 2023 +0000
@@ -65,7 +65,7 @@
                 [aResolver](const auto& aRv) { aResolver(ToNSResult(aRv)); });
 
             InvokeAsync(
-                dataManager->MutableIOTargetPtr(), __func__,
+                dataManager->MutableIOTaskQueuePtr(), __func__,
                 [dataManager =
                      RefPtr<fs::data::FileSystemDataManager>(dataManager),
                  rootId, parentEndpoint = std::move(parentEndpoint)]() mutable {